shared/src/models/simulated-region.ts

```diff
@@ -22,14 +22,20 @@ export class SimulatedRegion {
     public readonly type = 'simulatedRegion';
 
     /**
-     * top-left position
+     * Position of the viewport.
+     * For positive {@link size}s, this refers to the upper left corner of the simulated region, but during resizing, corners may be swapped.
```

Comment:
```suggestion
     * For positive {@link size}s, this refers to the upper left corner of the simulated region, but after resizing, corners may be swapped.
```

To make clear that this does not only happens *while* resizing (so in the act of it) but also persists thereafter.